portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -29,10 +28,12 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that returns some example content.*/
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
   
+  private int quantity;
```

Comment:
Please add comment explaining what this variable is. Also consider if it's neccessary for it to be a member variable or if it can be a local variable during the methods that it is used.

Reply:
I think making this a class variable will mean that during the lifetime of the servlet, every update of the limit will be updating this variable. So different users will be updating the same number. I wonder if there can be any race conditions.